src/modules/fundingManager/rebasing/abstracts/ElasticReceiptTokenBase_v1.sol

```diff
@@ -210,7 +210,7 @@ abstract contract ElasticReceiptTokenBase_v1 is IRebasingERC20, ERC165 {
     /// @dev The supply target MUST never be zero or higher than MAX_SUPPLY,
     ///      otherwise no supply adjustment can be executed.
     /// @dev Has to be implemented in downstream contract.
-    function _supplyTarget() internal virtual returns (uint);
+    function _supplyTarget() internal view virtual returns (uint);
```

Comment:
This change was needed as `balanceOf()` was complaining otherwise (as it's a `view` function). Should be a `view` function anyways, so this is fine.